### ISSUE TITLE
💡 RFC: astro serve command 

### ISSUE THREAD

**Issue by IanVS** [snowpackjs/astro#914]
### Background & Motivation

I'd love to be able to easily check my production bundle locally, without needing to manually spin up a separate http server for the bundle built with `astro build`.  

### Possible solutions

Add an `astro serve` command to the CLI to serve up the assets in the build directory, similar to what vite does.

### Help make it happen!

- [ ] I am willing to submit a PR to implement this change.
- [ ] I am willing to submit a PR to implement this change, but would need some guidance.
- [X] I am not willing to submit a PR to implement this change.

**Comment by eyelidlessness** [comment]
This is a great idea, **however** I’d caution against a naive solution. I’ve tried basically all of the reputable off-the-shelf static server libraries I could find on NPM and they all have caveats that may misrepresent your actual build’s perf characteristics. Many of them aggressively set cache headers that don’t reflect reality and cause subsequent builds to load… wrong (especially in Safari, which is a whole other thing, you basically have to dump your whole cache on every build to be sure you’re seeing the real build).

I eventually retreated to Vercel’s `serve`, which has the opposite problem where everything it serves never hits the cache at all (again, except for Safari 🤯).

**Comment by IanVS** [comment]
It would be pretty sweet if this is implemented in snowpack and then used by astro, so that snowpack users get this functionality as well.  That was a feature I was missing in Snowpack when I was trying it out directly.

**Comment by jasikpark** [comment]
'snowpack serve' = 'snowpack build' and then 'serve dist/'?

**Comment by IanVS** [comment]
Yes, or I guess `snowpack preview` if that's the command name that astro wants to use.

**Comment by FredKSchott** [comment]
> It would be pretty sweet if this is implemented in snowpack and then used by astro, so that snowpack users get this functionality as well. That was a feature I was missing in Snowpack when I was trying it out directly.

+1, agreed
